src/components/Header.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <header class="header">
+    <h1 class="header__city">Погода у {{ currentPosition.city }}</h1>
+    <div class="header__theme-toggle" @click.prevent="toggleTheme">
+      <svg
```

Comment:
[Optional] I would put svg into `assets/images` 
For instance using this loader
https://www.npmjs.com/package/vue-svg-loader

Reply:
@probil, should I use this in other components?

Reply:
@liubich Why not?

Reply:
@probil, looks like in this case I should make my components regular "not functional"